modules/fru.c

```diff
@@ -225,6 +225,9 @@ size_t fru_read( uint8_t id, uint8_t *rx_buff, uint16_t offset, size_t len )
         return 0;
     }
 
+    /*
+     * Read runtime FRU info
+     */
```

Comment:
I would improve this comment to mention that the 'runtime' FRU is auto-generated when there is no valid FRU info in the EEPROM.